packages/core/test/insured-bridge/e2e/InsuredBridge.e2e.js

```diff
@@ -23,13 +23,16 @@ const {
   PROXY__OVM_L1_CROSS_DOMAIN_MESSENGER,
 } = require("./helpers/OptimismConstants");
 
+const chainId = "10";
```

Comment:
did you actually run these tests out of interest?

Reply:
Also, I cant wait until the optimism compiler is updated. We can ditch a lot of the painful docker stuff.

we'll also want to have different integration tests for each L2 to ensure sufficient coverage.

Reply:
No I haven't, I'm not sure how to actually run these tests